common/reviews/api/node-core-library.api.ts

```diff
@@ -55,6 +139,16 @@ enum FolderConstants {
   NodeModules = "node_modules"
 }
 
+// @beta (undocumented)
+interface IColorableSequence {
+  // (undocumented)
+  backgroundColor?: ColorValue;
+  // (undocumented)
+  foregroundColor?: ColorValue;
+  // (undocumented)
+  text: string;
+}
+
```

Comment:
Maybe we should make this opaque.  Then you wouldn't need to export ColorValue at all.

Try using a symbol.  Hopefully API Extractor 6 won't crash.  :-P